src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/GetNextTask_2.sql

```diff
@@ -1,83 +0,0 @@
-﻿/*************************************************************
-    Stored procedures for get next available task
-**************************************************************/
---
--- STORED PROCEDURE
---     GetNextTask
---
--- DESCRIPTION
---     Get next available tasks
---
--- PARAMETERS
---     @queueId
---         * The ID of the task record
---     @count
---         * Batch count for tasks list
---     @taskHeartbeatTimeoutThresholdInSeconds
---         * Timeout threshold in seconds for heart keep alive
-CREATE PROCEDURE [dbo].[GetNextTask_2]
```

Comment:
I expected operation "Edit, Ren". Like on ResetTask sql. I don't think Delete should be here.

Reply:
I have checkout the GetNextTask_2.sql from main and then change its name to 3, but it still shows delete + add rather rename, is there any good solutions to do this?

Reply:
You shoud try edit/rename.